### DISCUSSION TITLE
<title>Help for a novice

### DISCUSSION THREAD

**Discussion by FrancoisPapin** [community/community#46323] [Actions]
### Select Topic Area

Product Feedback

### Body

My page "https://github.com/FrancoisPapin/NPEi.lab" is down because githubpages failures.  So, actually, some code changes don't induce effective website changes.
 
How could I fix the error?
How could I activate and modify the website related to codes ?

Please, maybe, someone could help me to resolve this bug?

**Comment by airtower-luna** [comment]
The builds seem to be crashing because they're out of disk space: https://github.com/FrancoisPapin/NPEi.lab/actions/runs/4087996785

Your repository seems to contain a lot of large PDFs, it might simply be too big.
